### DISCUSSION TITLE
Favlist save&load

### DISCUSSION THREAD

**Discussion by alien299** [hugoattal/OrbitalMarket#43] [General]
Hi!
Since the favlist is stored in local storage in my browser, is there any way I can save my current favlist and load it on an another computer? 
Does it work if I just copy the favlist local storage and store it as a txt? 
Like:
0: "657a78f6369472fc0ab1e7c1"
1: "650e5a8f24bd5e1a09490378"

**Comment by UncleFred** [comment]
I second this, it would be great to save these lists.

**Comment by hugoattal** [comment]
Done!
![image](https://github.com/hugoattal/OrbitalMarket/assets/4563971/6e9562f7-c0da-496c-a1ea-e5a4400f7a99)

Here's an example on an export:
```
60135243eb17c120c6ef5b25,60135394eb17c120c6efc925,601352b9eb17c120c6ef8627
```

The `import` ADD the imported element to the favlist, it does not replace all the existing fav elements.

**Reply by UncleFred** [reply]
Thank you for the update Hugo. This looks great.
